### PR TITLE
Add recommendations, refine header typography

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

### CODE REVIEW

File: content/word-list.md [hunk 1]
```diff
@@ -1,3 +1,99 @@
...
+**Proposed alternative:** 
+
+* allowlist/denylist 
+* allowedNouns/deniedNouns
+
+**Reasoning:** 
+
+The underlying assumption of the whitelist/blacklist metaphor is that white = good and black = bad. 
+Because colors in and of themselves have no predetermined meaning, any meaning we assign to them is cultural: for example, the color red in many Southeast Asian countries is lucky, and is often associated with events like marriages, whereas the color white carries the same connotations in many European countries. 
+In the case of whitelist/blacklist, the terms originate in the publishing industry – one dominated by America and England, two countries which participated in slavery and which grapple with their racist legacies to this day.
```
Comment (xmulligan) [inline content/word-list.md:20]:
```suggestion
In the case of whitelist/blacklist, the terms originate in the publishing industry – one dominated by the USA and England, two countries which participated in slavery and which grapple with their racist legacies to this day.
```

File: content/word-list.md [hunk 2]
```diff
@@ -1,3 +1,99 @@
...
+## Master, slave
+
+**Definition(s):**
+
+The “master/slave” metaphor in computing refers to a system with a single store of record and a set of replicas which maintain copies of the original data.
+
+**Proposed alternatives:**
+
+These recommendations are broken into two groups. 
+We recommend choosing an alternative that does not replace one metaphor with another, but understand that some projects may feel that Leader/follower or Parent/child communicates best.
```
Comment (xmulligan) [inline content/word-list.md:43]:
```suggestion
These recommendations are broken into two groups. The preferred group does not use a new metaphor while the also acceptable group does. We recommend choosing an alternative that does not replace one metaphor with another, but understand that some projects may feel that Leader/follower or Parent/child communicates best.
```

File: content/word-list.md [hunk 3]
```diff
@@ -1,3 +1,99 @@
...
+
+**Reasoning:**
+
+As the IETF puts it, “Master-slave is an oppressive metaphor that will and should never become fully detached from history.” 
+The word’s origins and historicial use reveal use that is at best chauvinistic and racist, and in almost all cases connotative of ownership. 
+While there is some small ambiguity about the term master, the term slave is unambiguously about the ownership and subjugation of another person, and has been since its inception. 
+
+The terms master/slave are harmful to Black and people of color contributors and employees. 
+Slavery is a tradition barely 3 generations abolished – there are grandparents alive today who were actual, non-metaphorical slaves. 
+Segregation is even more recent.
```
Comment (xmulligan) [inline content/word-list.md:65]:
```suggestion
Segregation and Apartheid are even more recent. 
```

File: content/word-list.md [hunk 4]
```diff
@@ -1,3 +1,99 @@
...
+**Reasoning:**
+
+As the IETF puts it, “Master-slave is an oppressive metaphor that will and should never become fully detached from history.” 
+The word’s origins and historicial use reveal use that is at best chauvinistic and racist, and in almost all cases connotative of ownership. 
+While there is some small ambiguity about the term master, the term slave is unambiguously about the ownership and subjugation of another person, and has been since its inception. 
+
+The terms master/slave are harmful to Black and people of color contributors and employees. 
+Slavery is a tradition barely 3 generations abolished – there are grandparents alive today who were actual, non-metaphorical slaves. 
+Segregation is even more recent. 
+In accordance with most open source codes of conduct and company handbooks, the mandate of all people in a project is to create welcome space, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
```
Comment (xmulligan) [inline content/word-list.md:66]:
```suggestion
In accordance with most open source codes of conduct and company handbooks, the mandate of all people in a project is to create a welcoming space, regardless of the level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality. 
```

File: content/word-list.md (hunk 3 above)
Comment (celestehorgan) [reply content/word-list.md:65]: yeeee! 🙌

**Pull request merged**